encrypted-media-respec.html

```diff
@@ -537,7 +537,18 @@
           </p>    
           <p>Time is intended to be consistent with Unix time.   
 		  </p>
-        </dd> 
+        </dd>
+        <dt id="user-agent-profile">User Agent Profile</dt>
```

Comment:
Since this includes profiles in different OS user accounts, "browsing profile" might make more sense since it is not necessarily a feature of the UA.